builtin/logical/aws/path_roles.go

```diff
@@ -80,6 +80,15 @@ will be passed in as the Policy parameter to the AssumeRole or
 GetFederationToken API call, acting as a filter on permissions available.`,
 			},
 
+			"iam_groups": &framework.FieldSchema{
+				Type: framework.TypeCommaStringSlice,
+				Description: `Names of IAM groups that generated IAM users will be added to. For a credential
+type of assumed_role or federation_token, the policies sent to the
+corresponding AWS call (sts:AssumeRole or sts:GetFederation) will be the
+policies from each group in iam_groups combined with the policy_document
+and policy_arns parameters.`,
+			},
```

Comment:
I know that on the other (older) parameters, there's nothing but `Type` and `Description`. However, since their creation, multiple new fields have been added to the `framework.FieldSchema` object to help UI users.

Could we add the `DisplayAttrs` field here? Particularly, adding a `Name` and a sample `Value` could be valuable to users who are trying to figure out how exactly to format the given string. The rest of the `FieldSchema` parameters don't seem as relevant.

Reply:
Oh that's cool. Added in 64f2dd206ed1b68bac70ca7bc7f1cfa6a54c8325